### ISSUE TITLE
about dist_loss

### ISSUE THREAD

**Issue by PeisongCheng** [harryhan618/LaneNet#7]
When read the code, I found the dist_loss may be have some mistakes,
dist_loss = dist_loss + torch.sum(F.relu(dist - self.delta_d)**2) / (num_lanes * (num_lanes-1)) / 2

I think it should be 
dist_loss = dist_loss + torch.sum(F.relu(self.delta_d - dist)**2) / (num_lanes * (num_lanes-1)) / 2

but i am not sure, thank you!

**Comment by harryhan618** [comment]
Yes, you are right. I just revised the code yesterday. You can git pull the code.
Also, in the paper, it seems that `reg_loss` isn't used. 
I'm training the model these days. So plz keep update.

**Comment by harryhan618** [comment]
@PeisongCheng Excuse me, there is a bug in the `discriminative_loss`. In my code, `dist` is a matrix indicating 2-norm of pairwise centriods' difference. Thus the diagonal elements are 0 (norm of difference of same centriod). They shouldn't get counted into loss. This bug is fixed.
